projects/plugins/jetpack/_inc/client/newsletter/email-settings.jsx

```diff
@@ -64,7 +79,32 @@ const EmailSettings = props => {
 		disabled || isSavingAnyOption( [ SUBSCRIPTION_EMAILS_USE_EXCERPT_OPTION ] );
 
 	const replyToInputDisabled = disabled || isSavingAnyOption( [ REPLY_TO_OPTION ] );
+	const fromNameInputDisabled = disabled || isSavingAnyOption( [ FROM_NAME_OPTION ] );
 
+	const [ fromNameState, setFromNameState ] = useState( {
+		value: subscriptionFromName,
+		hasChanged: false,
+	} );
+
+	const handleSubscriptionFromNameChange = useCallback(
+		event => {
+			setFromNameState( {
+				value: event.target.value,
+				hasChanged: subscriptionFromName !== event.target.value,
+			} );
+		},
+		[ setFromNameState, subscriptionFromName ]
+	);
+
+	const handleSubscriptionFromNameChangeClick = useCallback( () => {
+		updateFormStateAndSaveOptionValue( FROM_NAME_OPTION, fromNameState.value );
+		analytics.tracks.recordEvent( 'jetpack_newsletter_set_reply_to_name', {
+			value: fromNameState.value,
+		} );
+		setFromNameState( { value: fromNameState.value, hasChanged: false } );
+	}, [ fromNameState, updateFormStateAndSaveOptionValue ] );
+	const exampleEmail =
+		subscriptionReplyTo !== 'author' ? 'noreply@wordpress.com' : 'author-name@example.com';
```

Comment:
`noreply` -> `donotreply` is the correct email I believe

Reply:
Good catch!